doc/manual/migrating_from_eeglab.rst

```diff
@@ -0,0 +1,68 @@
+* does not have an equivalent to EEGLAB's *STUDY* structure. Grand averages of single-subject `evoked` and `average_tfr` objects can be constructed with the `mne.grand_average` function (beware of the default setting of weighting by trials), which return regular `evoked` and `average_tfr` objects. Individual subjects can simply be stored in lists or dicts of `epochs` or `raw` objects.
+
+Partial EEGLAB/MNE translation table
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+============================================================================================================================
+                              Software
+Concept                       EEGLAB          			        MNE
+Importing data		`EEG = pop_fileio('/your/data.dat');`		raw = mne.io.read_raw('/your/data-raw.fif')
+
+Filtering raw data	`EEG = pop_eegfiltnew(EEG, 1, 40);`			raw.filter(1, 40)
+
+Run ICA				`EEG = pop_runica(EEG);`					ica = mne.preprocessing.ica.ica().fit(raw)
+
+
+Epoching data		`EEG_epochs = pop_epoch(EEG,				`epochs = mne.Epochs(raw,
+																events=mne.find_events(raw),
+                              {'1', '2'},		                event_id=dict(cond1=1, cond2=2),
+                              [-.2, .8]);`						tmin=-2, tmax=.8)`
+
+Selecting epochs    `EEG_epochs2 = pop_epoch(EEG_epochs,		`epochs_2 = epochs["cond2"]`
+                            {'2'});`
+
+
+
+ERPs															`evoked = epochs.average()`
+
+Plot Butterfly ERP	`pop_timtopo(EEG_epochs, ...);`             `evoked.jointplot()`
+
+Contrast ERPs		`pop_compareerps(EEG_epochs, EEG_epochs2);`	`(evoked - evoked2).plot()`
+
+Saving data         `EEG = pop_saveset(EEG,'filename',set);`	`[raw/epochs/evoked].save('/your/data-[raw/epo/evo].fif')
+
+============================================================================================================================
+
+
+Tips on specific features for people coming from EEGLAB
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* The MNE event system stores onset, duration, and type (as integers). Beware of the find_events function; with its default settings, it will not find many events in typical EEG formats. Try using `shortest_event=0, min_duration=0, consecutive=True` if that happens to you.
+* The MNE resampling function is much slower than the EEGLAB equivalent. As a temporary work-around, many MNE functions allow a `decim` keyword to downsize analyses or objects. Also, remember that MNE can be more readily parallelized than MNE; via e.g. joblib, a whole experiment can be downsampled simultaneously.
+* The MNE `eog` channel type is generally conceptualized around bipolar eye channels. Channels marked as `eog` are not included in many analyses. Typical EEGLAB behavior is roughly equivalent to setting all EEG channels, including monopolar eye sensors, to channel type `eeg`.
```

Comment:
I would be more explicit. Can you put in two lines of code here? Something like:

```py
In contrast to EEGLAB, you are required to specify the EOG channels in MNE as it is critical to many analysis steps including preprocessing, projection and forward calculation. This can be done when reading in the file:

raw = read_raw_eeglab('filename.set', eog=['EOG1, 'EOG2'])
```